### ISSUE TITLE
Is GitHub CLI a dependency?

### ISSUE THREAD

**Issue by AliOsm** [Shipyrd/shipyrd#28]
Running `Shipyrd::Client.new.trigger('pre-connect')` manually from `rails c` shows:

```
INFO -- Shipyrd: pre-connect trigger failed with error => No such file or directory - gh
```

Should I install GitHub CLI on my machine?

**Comment by AliOsm** [comment]
I prefer to not force GitHub CLI, if it exists, then Shipyrd can use it, if it is not installed, then either use `ENV["KAMAL_PERFORMER"]`, or the better option imo is to use the name/email address associated with the commit.

**Comment by nickhammond** [comment]
@AliOsm Ah, good catch. I'll have it fail gracefully if `gh` isn't available. The intention is for it to use `gh` details if available and then if not fallback to `ENV["KAMAL_PERFORMER"]`. 

Right now I just have GitHub support in there which then helps with loading the avatar within Shipyrd. There isn't a UI to edit user details at the moment so we'd lose the avatar and probably put a placeholder letter-based avatar in there.

**Comment by nickhammond** [comment]
The `gh` CLI dependency issue is fixed in [v0.2.7](https://github.com/Shipyrd/shipyrd-gem/releases/tag/v0.2.7) for the gem, still going to work on adding additional name/email/username support in another PR though.
